packages/babel-compat-data/data/plugin-bugfixes.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
@nicolo-ribaudo - meta-comment, more on the data format itself: do you think it might be easier to maintain this file if the data only specified which versions were actually affected by the bugfix plugins?  For example, the edge function name plugin could just be listed as `{ "edge": "79" }` rather than including the other browser versions which it doesn't have an effect on. A missing browser version could just be taken as "no change".

Reply:
This data is generated by the same script which generated the old data, so it's easier to generate all the targets :sweat_smile:

Reply:
Ah! gotcha.

Reply:
it could be interesting to create a rendered version based on the data on our website or something? Or just showcasing this in general in a more readable manner?

Reply:
Something like compat-table but with plugin names rather than ECMAScript features?

Reply:
Yeah, then you could generate what @developit was saying allow about seeing the diff's or what was actually affected? Maybe we should add a simple readme to explain each json file